Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1508,6 +1510,25 @@ inline FontVariationSettings BuilderConverter::convertFontVariationSettings(Buil
     return settings;
 }
 
+inline std::optional<FontSizeAdjust> BuilderConverter::convertFontSizeAdjust(BuilderState&, const CSSValue& value)
+{
+    if (is<CSSPrimitiveValue>(value)) {
+        auto& primitiveValue = downcast<CSSPrimitiveValue>(value);
+        if (primitiveValue.valueID() == CSSValueNone)
+            return std::nullopt;
+
+        ASSERT(primitiveValue.isNumber());
+        FontSizeAdjust fontSizeAdjust = { FontSizeAdjust::Metric::ExHeight, primitiveValue.floatValue() };
+        return fontSizeAdjust;
+    }
+
+    ASSERT(value.isPair());
+    const auto& pair = downcast<CSSValuePair>(value);
+    FontSizeAdjust fontSizeAdjust = { fromCSSValueID<FontSizeAdjust::Metric>(downcast<CSSPrimitiveValue>(pair.first()).valueID()), downcast<CSSPrimitiveValue>(pair.second()).floatValue() };
```

Comment:
Nit: for clarity I'd prefer to pull out `.first()` and `.second()` onto previous lines and give them variable names.

Something like:
```
auto metric = fromCSSValueID<FontSizeAdjust::Metric>(downcast<CSSPrimitiveValue>(pair.first()).valueID());
auto value = downcast<CSSPrimitiveValue>(pair.second()).floatValue();
FontSizeAdjust fontSizeAdjust { WTFMove(metric), WTFMove(value) };
```

Reply:
O.K.